com.microsoft.mrtk.input/Interactors/GazePinch/GazePinchInteractor.cs

```diff
@@ -170,6 +173,18 @@ private void ComputeAttachTransform(IXRSelectInteractable interactable)
             {
                 rotationToApply = PlayspaceUtilities.ReferenceTransform.rotation * device.deviceRotation.ReadValue();
             }
+            else
+            {
+                // If we don't have a valid device pose, let's use the palm pose; closest thing we've got!
+                if (handsAggregator == null)
+                {
+                    handsAggregator = HandsUtils.GetSubsystem();
+                }
+                if (handsAggregator != null && handsAggregator.TryGetJoint(TrackedHandJoint.Palm, handController.HandNode, out HandJointPose palmPose))
```

Comment:
We could use handsAggregator.TryGetHandCenter(hand, out jointPose) here instead. I think its more appropriate here than specifically targetting the palm joint, which isn't a "real" joint. It also matches what our Simulated Controller uses.

Reply:
Sure, I'll look into this

Reply:
Why not palm, again? If we're trying to match ray-like behavior, the ray algorithm actually specifically states that the input normal is a _palm_ normal.